go.mod

```diff
@@ -185,3 +186,5 @@ require (
 	gopkg.in/yaml.v2 v2.4.0 // indirect
 	mvdan.cc/lint v0.0.0-20170908181259-adc824a0674b // indirect
 )
+
+require golang.org/x/net v0.0.0-20220722155237-a158d28d115b
```

Comment:
I'm not a maintainer of this code, but I assume that something is off when you have a list of requirements at the same time as you have a one-off require statement. Removing this line and running `go mod tidy` should fix everything.

Reply:
Sure, it's done.